<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="txt_content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_result">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_sin">
        <property name="text">
         <string>sin(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cos">
        <property name="text">
         <string>cos(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tan">
        <property name="text">
         <string>tan(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_log10">
        <property name="text">
         <string>log10(</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_asin">
        <property name="text">
         <string>asin(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_acos">
        <property name="text">
         <string>acos(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_atan">
        <property name="text">
         <string>atan(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_log">
        <property name="text">
         <string>log(</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_sinh">
        <property name="text">
         <string>sinh(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cosh">
        <property name="text">
         <string>cosh(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tanh">
        <property name="text">
         <string>tanh(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_PI">
        <property name="toolTip">
         <string>圆周率</string>
        </property>
        <property name="text">
         <string>PI</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_left_bracket">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_right_bracket">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_E">
        <property name="toolTip">
         <string>自然对数</string>
        </property>
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_sub">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btn_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mul">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btn_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_div">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="btn_backspace">
        <property name="toolTip">
         <string>后退</string>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dot">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pow">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;乘幂&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>**</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calu">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
